packages/calypso-typescript-config/ts-package.json

```diff
@@ -13,7 +13,7 @@
 		"noEmitHelpers": true,
 		"skipLibCheck": true,
 
-		"lib": [ "ES2020", "DOM", "DOM.Iterable" ],
+		"lib": [ "ES2022", "DOM", "DOM.Iterable" ],
 		"module": "ES2020",
 		"target": "ES2020",
```

Comment:
I think it'd be ok to update this to ES2022 as well

Reply:
FWIW the difference is well explained here:

https://stackoverflow.com/questions/67371787/what-typescript-configuration-produces-output-closest-to-node-js-16-capabilities/67371788#67371788